packages/apps-config/src/endpoints/productionRelayPolkadot.ts

```diff
@@ -425,6 +425,19 @@ export const prodParasPolkadot: Omit<EndpointOption, 'teleport'>[] = [
       logo: nodesMoonbeamSVG
     }
   },
+  {
+    homepage: 'https://moonsama.com',
+    info: 'moonsama',
+    paraId: 2199,
+    providers: {
+      'Moonsama': 'wss://rpc.moonsama.com/ws'
```

Comment:
```suggestion
      Moonsama: 'wss://rpc.moonsama.com/ws'
```

This should fix the linting 

(weirdly in the GH view it shows on the wrong line, aka 2 below)